static/docs/user-guide/contributing-documentation.md

```diff
@@ -78,22 +78,23 @@ $ source .env/bin/activate
 ```
 
 - Install the style checker's requirements using
-`pip install -r requirements.txt`.
+  `pip install -r requirements.txt`.
 
 - Install coding style pre-commit hook with `pre-commit install`.
 - Once the `pre-commit` hook is installed, you may deactivate the virtual
-environment by running `deactivate`.
+  environment by running `deactivate`.
 
 It's highly recommended to run the Node.js docs app locally to test significant
 changes to the docs before submitting them, and its very much needed in order to
 make changes to the docs JavaScript engine itself (rare). To do so, please
 follow the steps below:
 
 - Make sure you have the latest version of [Node.js](https://nodejs.org/en/)
-installed. Install dependencies by running the command `npm install`.
+  installed. Install and keep the dependencies up to date by running
+  `npm install` often.
```

Comment:
Note: I updated the sentence above per #363 

All other changes to this file are by linting it with `npx prettier`.

Reply:
@shcheklein @jorgeorpinel is it needed to mention first run 
`npm install -g npm` first?
then run 'npm install'

Reply:
Having the latest Node.js installed is mentioned in the previous paragraph.